integrations/opencost/main.py

```diff
@@ -1,28 +1,11 @@
 from typing import Any
-from enum import StrEnum
-from port_ocean.context.ocean import ocean
-from client import OpenCostClient
-
-
-class ObjectKind(StrEnum):
-    COST = "cost"
-
 
-def init_client() -> OpenCostClient:
-    return OpenCostClient(
-        ocean.integration_config["app_host"],
-        ocean.integration_config.get("window", "today"),
-    )
-
-
-def process_cost_item(item: dict[str, Any]) -> list[dict[str, Any]]:
-    return [value for value in item.values()]
+from client import OpenCostClient
+from port_ocean.context.ocean import ocean
 
 
-@ocean.on_resync(ObjectKind.COST)
+@ocean.on_resync("cost")
```

Comment:
Why removing the enum?

Reply:
just remove extra code that wasnt needed to keep the code simple